lib/countries/data/subdivisions/AR.yaml

```diff
@@ -1802,4 +1813,10 @@ Z:
     yue: 聖古絲省
     mk: Санта Круз
     'no': Santa Cruz
+    az: Los Buzlaqları Milli Parkı
+    hr: Nacionalni park Los Glaciares
+    ml: ലോസ് ഗ്ലേസിയേഴ്സ് നാഷണൽ പാർക്ക്
+    my: လော့စ်ဂလက်ဆရားရက်စ် အမျိုးသားဥယျာဉ်
+    ne: लोस ग्लासिआरेस राष्ट्रिय निकुञ्ज
+    sq: Parku Kombëtar Los Glaciares
```

Comment:
This is an example of a locale which we don't have translations for in the country translations directory. As mentioned in the main comment I'm not sure whether we want a supported locale configuration or if we eventually want to include all translations.

Reply:
Which locale? I think we have these 6.